Dockerfile

```diff
@@ -24,7 +24,21 @@ RUN apk add \
   && \
   rm -rf /var/cache/apk/*
 
+WORKDIR /tmp
+RUN apk update && \
```

Comment:
Merge them to the before `RUN` script, with one line.

Reply:
Or just move the `libstdc++ gcompat icu`, and keep the `sqlpackage` install in split.

Reply:
I'll keep mssql installation in the separate layer. Thanks.